src/app/src/main/java/com/example/bookmark/AddBookActivity.java

```diff
@@ -45,5 +50,18 @@ private void doneAddBook() {
         // TODO: Call uriToPhotograph() only here, once the done button has been pressed (Uri to Bitmap is relatively expensive)
         // TODO: Create the book object, send it to Firebase?
```

Comment:
Remove this

Reply:
It's still technically not done because I didn't set the book's photograph. We can remove this comment when that's done.